shared/login/index.js

```diff
@@ -19,7 +19,7 @@ function loginRouter (currentPath: Map<string, string>, uri: URI): any {
 
   const path = currentPath.get('path')
 
-  const {component: Component, props} = currentPath.get('parseRoute') || {}
+  const {componentAtTop: {component: Component, props}} = currentPath.get('parseRoute') || {componentAtTop: {}}
```

Comment:
The login flow dynamically pushes routes with context so we send a
structure back. Look at the actions for login

On Tuesday, February 9, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In shared/login/index.js
> <https://github.com/keybase/client/pull/1939#discussion_r52392957>:
>
> > @@ -19,7 +19,7 @@ function loginRouter (currentPath: Map<string, string>, uri: URI): any {
> >
> >    const path = currentPath.get('path')
> >
> > -  const {component: Component, props} = currentPath.get('parseRoute') || {}
> > +  const {componentAtTop: {component: Component, props}} = currentPath.get('parseRoute') || {componentAtTop: {}}
>
> isn't the parseRoute a function?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1939/files#r52392957>.
>